README.rst

```diff
@@ -83,6 +83,33 @@ The following is a basic example of using the **nidmm** module to open a session
         session.configureMeasurementDigits(nidmm.Function.DC_VOLTS, 10, 5.5)
         print("Measurement: " + str(session.read()))
 
+Repeated capabilities
+---------------------
+
+* Repeated capabilities are accessed using a repeated capabilies object for each type of repeated capability that the driver supports. Some drivers do not use repeated capabilities.
+* The repeated capabilities object can take integers or strings, a single item or a list, or a slice.
+* The repeated capabilities object knows the proper prefix and will add it if needed.
+* The following are all legal
+
+    .. code-block:: python
+
+        import nifgen
+        session = nifgen.Session('PXI1Slot2')
```

Comment:
how come you didn't use `with`? If you'd still rather not, then maybe show how you close the session?